tools/mirror_repos_forgejo/clone_packages.py

```diff
@@ -0,0 +1,146 @@
+import os
+import toml
+import time
+import config
+import requests
+
+
+APPS_REPO_ROOT = config.APPS_REPO_ROOT
+GITHUB_TOKEN = config.GITHUB_TOKEN
+FORGEJO_TOKEN = config.FORGEJO_TOKEN
+
+catalog = toml.load(open(os.path.join(APPS_REPO_ROOT, "apps.toml"), encoding="utf-8"))
```

Comment:
(If later you want to iterate on the Yunohost-Apps org repo instead, I'd recommend using PyGithub which is way better than crafting request manually :wink: )